src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -3713,20 +3713,19 @@ void MacroAssembler::load_nklass(Register dst, Register src) {
   b(done);
 
   bind(slow);
-  enter();
```

Comment:
enter() saves/restores lr and rfp, and then overrides rfp with current sp. leave() reverts that. C2 doesn't seem to like messing with rfp, but we still need to preserve lr at this point, so that subroutine calls can find their way back. The stub does its own enter/leave around the slow-path call.

Reply:
OK.